content/articles/cancer-cell-classification-using-scikit-learn/index.md

```diff
@@ -0,0 +1,155 @@
+
+### Prerequisites
+1. Using Scikit-learn will be a necessity for our project. 
+2. Jupyter notebooks are preferred for this project, however you are free to use any IDE of your choice.
+
+### Table of contents
+- [Naive Bayes algorithm](#naive-bayes-algorithm)
+    - [Advantages of the Naive Bayes classification](#advantages-of-the-Naive-Bayes-classification)
+    - [Disadvantages of the Naive Bayes classification](#disadvantages-of-the-Naive-Bayes-classification)
+- [The dataset](#the-dataset)
+- [Classification implemented with the Scikit-learn framework](#classification-implemented-with-the-scikit-learn-framework)
+- [Conclusion](#conclusion)
+
+### Naive Bayes algorithm
+Despite its simplicity, Naive Bayes is a potent algorithm for predicting outcomes. In Naive Bayes, each input variable is assumed to be independent, which is why it's named naive. In reality, this is a big assumption, and the technique works well for a wide range of difficult tasks.
+
+This popular method for binary classification, dubbed Naive Bayes, will be selected for use in this model.
+
+### Advantages of the Naive Bayes classification
+- It's a snap to put into action.
+- Less training data is needed.
+- Continuous and discrete data are also supported.
+- The number of predictors and data points can be increased indefinitely, making it extremely flexible and adaptable.
+- It is rapid and can be used to make predictions in real-time.
+- 
+### Disadvantages of the Naive Bayes classification
+- In naive Bayes, all predictors (or traits) are assumed to be independent, however, this is rarely the case in reality.
+- To avoid assigning zero probability to a categorical variable whose category was not accessible in the training dataset, this technique uses the 'zero-frequency problem.
+- 
+### Installations
+Run the following line on the command prompt to download and install it on your computer, or [follow this documentation.](https://scikit-learn.org/0.17/install.html)
+
+```bash
+pip install scikit-learn
+```
+> For this project, Jupyter notebooks are encouraged, although you are free to use any IDE you choose. Instead of developing an entire script from scratch, one can execute a few lines of code and see what happens one by one rather than writing the entire script and running it.
+
+To set it up, use the command line and enter the following code:
+
+```bash
+pip install jupyter
+```
+### The dataset
+We will be classifying cancer cells based on their features and identifying them if they are malignant or benign using scikit-learn library for python programming language. We will be using the breast cancer Wisconsin dataset for our machine learning problem.

```

Comment:
Please provide a link to the Breast Cancer Wisconsin dataset. Besides, you need to add the dataset you'll be using on your prerequisites section

Reply:
Why did you mark this as resolved yet you haven't provided the link?